sdk/schemaregistry/ci.yml

```diff
@@ -0,0 +1,31 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/template/
```

Comment:
this should say schemaregistry rather than template.

Reply:
Fixed.